### ISSUE TITLE
Global Autotype not working on Chrome tabs containing square brackets

### ISSUE THREAD

**Issue by ntodorov70** [MacPass/MacPass#1284]
**Describe the bug**
Global Autotype works well in Chrome, but doesn't work on tabs containing square brackets in the name (e.g. "Sing in [Jenkins]")

**To Reproduce**
1. Go to 'page containing square brackets in the title (appearing in the chrome tab)'
2. Click on 'global autotype shortcut e.g. ⌃⌥M'
3. Nothing happens, no username is typed in

**Expected behavior**
The user and password corresponding to the tab/window name get typed automatically in the entry fields 

**Desktop (please complete the following information):**
 - macOS: [13.6]
 - Version [0.8.1]

**Comment by mstarke** [comment]
That’s an interesting issue. I’ll try to get to the bottom of it. My first guess is some regex issue but I’m fairly sure I do escape the query but who knows. Nice find!

**Comment by ntodorov70** [comment]
Yes, regex issue was my first tought as well. I hope it can be fixed easily.


Michael Starke ***@***.***> schrieb am Sa., 21. Okt. 2023,
14:51:

**Comment by georgesnow** [comment]
If you have the urls stored in the entries you could use the URL Autotype plugin as a stopgap (or permanently).
